mne/inverse_sparse/mxne_inverse.py

```diff
@@ -700,3 +731,153 @@ def tf_mixed_norm(evoked, forward, noise_cov,
         out = out, residual
 
     return out
+
+
+@verbose
+def _compute_mxne_sure(M, gain, alpha_grid, sigma, n_mxne_iter, maxit, tol,
+                       n_orient, active_set_size, debias, solver, dgap_freq,
+                       random_state, verbose):
+    """Stein Unbiased Risk Estimator (SURE).
+
+    Implements the finite-difference Monte-Carlo approximation
+    of the SURE for Multi-Task LASSO.
+
+    See reference :footcite:`DeledalleEtAl2014`.
+
+    Parameters
+    ----------
+    M : array, shape (n_sensors, n_times)
+        The data.
+    gain : array, shape (n_sensors, n_dipoles)
+        The gain matrix a.k.a. lead field.
+    alpha_grid : array, shape (n_alphas)
+        The grid of alphas used to evaluate the SURE.
+    sigma : float
+        The true or estimated noise level in the data. Usually 1 if the data
+        has been previously whitened using MNE whitener.
+    n_mxne_iter : int
+        The number of MxNE iterations. If > 1, iterative reweighting is
+        applied.
+    maxit : int
+        Maximum number of iterations.
+    tol : float
+        Tolerance parameter.
+    n_orient : int
+        The number of orientation (1 : fixed or 3 : free or loose).
+    active_set_size : int
+        Size of active set increase at each iteration.
+    debias : bool
+        Debias source estimates.
+    solver : 'prox' | 'cd' | 'bcd' | 'auto'
+        The algorithm to use for the optimization.
+    dgap_freq : int or np.inf
+        The duality gap is evaluated every dgap_freq iterations.
+    random_state : int | None
+        The random state used in a random number generator for delta and
+        epsilon used for the SURE computation.
+
+    Returns
+    -------
+    X : array, shape (n_active, n_times)
+        Coefficient matrix.
+    active_set : array, shape (n_dipoles,)
+        Array of indices of non-zero coefficients.
+    best_alpha_ : float
+        Alpha that minimizes the SURE.
+
+    References
+    ----------
+    .. footbibliography::
+    """
+    def g(w):
+        return np.sqrt(np.sqrt(groups_norm2(w.copy(), n_orient)))
+
+    def gprime(w):
+        return 2. * np.repeat(g(w), n_orient).ravel()
+
+    def _run_solver(alpha, M, n_mxne_iter, as_init=None, X_init=None,
+                    w_init=None):
+        if n_mxne_iter == 1:
+            X, active_set, _ = mixed_norm_solver(
+                M, gain, alpha, maxit=maxit, tol=tol,
+                active_set_size=active_set_size, n_orient=n_orient,
+                debias=False, solver=solver, dgap_freq=dgap_freq,
+                active_set_init=as_init, X_init=X_init, verbose=False)
+        else:
+            X, active_set, _ = iterative_mixed_norm_solver(
+                M, gain, alpha, n_mxne_iter, maxit=maxit, tol=tol,
+                n_orient=n_orient, active_set_size=active_set_size,
+                debias=False, solver=solver, dgap_freq=dgap_freq,
```

Comment:
you pass debias in the function signature but you set it to False here

as I am not sure what debias does here I would suggest not to expose debias in the function _compute_mxne_sure

check however that debias is used somewhere if you use sure.

Reply:
debias is actually never used with SURE. I've changed from debias=False to debias=debias and kept debits in the function signature. All tests pass. Do you agree?